editor/editor_node.cpp

```diff
@@ -3706,12 +3706,17 @@ void EditorNode::_remove_scene(int index, bool p_change_tab) {
 }
 
 void EditorNode::set_edited_scene(Node *p_scene) {
+	set_edited_scene_root(p_scene, true);
+}
+
+void EditorNode::set_edited_scene_root(Node *p_scene, bool p_auto_add) {
+	ERR_FAIL_COND_MSG(p_scene && p_scene->get_parent(), "Not null nodes that are set as edited scene should not have a parent node.");
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(p_scene && p_scene->get_parent(), "Non-null nodes that are set as edited scene should not have a parent node.");
```

Reply:
You applied it wrongly 🙃

Reply:
In addition, should re-setting be allowed here? `p_scene == old_edited_scene_root`